automatic/smartftp/smartftp.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>{{PackageName}}</id>
+    <title>SmartFTP</title>
+    <version>{{PackageVersion}}</version>
+    <authors>SmartSoft</authors>
+    <summary>FTP, FTPS, SFTP, HTTP, Amazon S3, WebDAV, Google Drive, OneDrive, Telnet, Terminal client.</summary>
+    <description>FTP, FTPS, SFTP, HTTP, Amazon S3, WebDAV, Google Drive, OneDrive, Telnet, Terminal client.</description>
```

Comment:
@matbech typically the description is a paragraph that describes the package, rather than a list of tags.  Can you correct?